include/EASTL/sort.h

```diff
@@ -213,6 +213,54 @@ namespace eastl
 	/// inner for loop with a call to upper_bound, which would be faster.
 	///
 	template <typename BidirectionalIterator, typename StrictWeakOrdering>
+	void insertion_sort(
+	    BidirectionalIterator first,
+	    BidirectionalIterator last,
+	    StrictWeakOrdering compare,
+	    typename eastl::enable_if<
+	        eastl::is_move_constructible<typename eastl::iterator_traits<BidirectionalIterator>::value_type>::value>::
```

Comment:
check for is_move_assignable as well... as the  function body uses move assignment as well.